src/main/java/com/backbase/ct/bbfuel/input/TransactionsReader.java

```diff
@@ -50,6 +50,31 @@ public TransactionsPostRequestBody loadSingle(String externalArrangementId) {
         return list;
     }
 
+    /**
+     * Load transactions with reference filled with pocket parent external arrangement id.
+     *
+     * @param currentAccountExternalArrangementId current account external arrangement id
+     * @param parentPocketExternalArrangementId   parent pocket external arrangement id
+     * @return list of TransactionsPostRequestBody's
+     */
+    public List<TransactionsPostRequestBody> loadWithPocketParentAsReference(String currentAccountExternalArrangementId,
+        String parentPocketExternalArrangementId) {
+        List<TransactionsPostRequestBody> list = load(
+            globalProperties.getString(CommonConstants.PROPERTY_CURRENTACCOUNT_TRANSACTIONS_DATA_JSON));
+        list.forEach(transactionsPostRequestBody -> {
+            BigDecimal amount = CommonHelpers.generateRandomAmountInRange(1L, 200L);
```

Comment:
Amount needs to match what's ingested for pockets, it cannot be random.

Reply:
Good you see this, I forgot to remove those lines!